src/index.html

```diff
@@ -9,10 +9,65 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="../styles/index.css"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400..900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="img"
+        src="./images/imac.jpeg"
+        alt="imac"
+      />
+
+      <h3 class="discribe">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+      <p class="code">Product code: 195434</p>
+
+      <div class="rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="reviews">Reviews: 5</p>
+      </div>
+
+      <div class="price">
+        <p class="price">Price:</p>
+        <h2>$2,199</h2>
+      </div>
+
+      <button
+        class="buy-button"
+        data-qa="hover"
+      >
+        <a
+          href="#"
+          data-qa="hover"
+        >
+          Buy
+        </a>
+      </button>
```

Comment:
Using an anchor tag inside a button element is not semantically correct and can lead to accessibility issues. Consider using either a button element with a click event or an anchor tag styled as a button.